content/applications/sales/sales/products_prices/products/import.rst

```diff
@@ -1,43 +1,264 @@
+
+Once those errors are fixed, click :guilabel:`Test` again to ensure all necessary issues have been
+remedied appropriately.
+
+If additional product template spreadsheets need to be uploaded, click the :guilabel:`Load File`
+button, select the desired product template spreadsheet, and repeat the process.
+
+When everything is ready, click the :guilabel:`Import` button.
+
+When clicked, Odoo instantly imports those products, and reveals the main :guilabel:`Products` page,
+with a pop-up message in the upper-right corner. This pop-up message informs the user how many
+products were just successfully imported.
+
+.. image:: import/successful-import-popup.png
+   :align: center
+   :alt: The pop-up window that appears after a successful product import process in Odoo Sales.
+
+At this point, all the newly-imported products are accessible and editable via the
+:guilabel:`Products` page.
+
+Import relation fields, attributes, and variants
+================================================
+
+It's important to note that an Odoo object is always related to many other objects. For example, a
+product is linked to product categories, attributes, vendors, and things of this nature. These
+links/connections are known as relations.
+
+It should be noted that, in order to import product relations, the records of the related object
+**must** be imported *first* from their own list menu.
+
+Relation fields
+---------------
+
+On product forms in Odoo, there are a number of fields that can be modified and customized at any
+time. These fields are found under every tab on a product form. While these fields are easily
+editable directly on the product form, they can also be modified via a product import.
+
+As mentioned, relation fields of this nature can **only** be imported for products if they already
+exist in the database. For example, if a user attempts to import a product with a *Product Type*, it
+can only be one of the preconfigured product types existing in the database (e.g. *Storable
+Product*, *Consumable*, etc.).
+
+To import information for a relation field on a product import template spreadsheet, add the name of
+the field as a column name/title on the spreadsheet. Then, on the appropriate product line, add the
+desired relation field option.
+
+When all desired relation field information has been entered, save the spreadsheet, and import it
+to the database, per the process mentioned above (:menuselection:`Sales app --> Products -->
+Products --> ⚙️ (gear) icon --> Import records --> Upload File`).
+
+Once the spreadsheet with the newly-configured relation field information has been uploaded, click
+:guilabel:`Import`, and Odoo returns to the :guilabel:`Products` page.
+
+When the newly-changed/modified products (complete with the new relation field information) has been
+imported and uploaded, that new information can be found on that product's product page.
```

Comment:
```suggestion
imported and uploaded, that new information can be found on the :guilabel:`Products` page.
```
I understand you are trying to refer to an individual product record (page) here, so I do not know if this is the best wording, but I don't think "product's product page" works. Maybe just "Product record"?